src/Steam.package/STMongoStoreTest.class/instance/testGameModification.st

```diff
@@ -0,0 +1,12 @@
+tests
+testGameModification
+	| extractedCollection |
+	game rating: 10.
+	store save: game.
+	extractedCollection := store query: game class with: game title.
+	self assert: extractedCollection first rating equals: 10.
+	game rating: 5.
+	extractedCollection := store query: game class with: game title.
+	self assert: extractedCollection first rating equals: 5.
+	store delete: extractedCollection first	;
+		delete: gameCat
```

Comment:
You see, if we had a drop database in the tear down, you wouldn't need to delete the created objects at the middle of each test.

This also means that if an assertion fails, your database will remain dirty!

Reply:
You can already extract all the 

```
store
    delete: extractedCollection first;
    delete: gameCat 
```
Or similar into a `cleanupDatabase` method or so, no?